src/discussions/comments/data/selectors.js

```diff
@@ -8,3 +8,5 @@ export const selectCommentResponses = commentId => state => (state.comments.comm
 );
 
 export const commentsStatus = state => state.comments.status;
+
+export const courseSettingsSchemeDivided = state => state.courseSettings?.divisionScheme !== 'none';
```

Comment:
You are looking in the wrong place for these settings, so this will always return true. This setting should default to false. So you should use a logic like:

```javascript
divisionsScheme && divisionsScheme!=='none'
```

```suggestion
export const courseSettingsSchemeDivided = state => state.comments.courseSettings?.divisionScheme !== 'none';
```